evalml/automl/automl_algorithm/iterative_algorithm.py

```diff
@@ -62,8 +62,11 @@ def next_batch(self):
 
         next_batch = []
         if self._batch_number == 0:
-            next_batch = [pipeline_class(parameters=self._transform_parameters(pipeline_class, {}), random_seed=self.random_seed)
-                          for pipeline_class in self.allowed_pipelines]
+
+            # next_batch = [pipeline_class(parameters=self._transform_parameters(pipeline_class, {}), random_seed=self.random_seed)
+            #               for pipeline_class in self.allowed_pipelines]
+            next_batch = [pipeline.new(parameters=self._transform_parameters(pipeline, self._pipeline_params), random_seed=self.random_seed)
```

Comment:
TODO: how to pass pipeline params to first pipelines... Probs need pipelines to just have..?